build.gradle

```diff
@@ -186,6 +185,7 @@ def distContent = copySpec {
   into('build') {
     from "$buildDir"
     include '*.jar'
+    include 'i386/*.so'
```

Comment:
Yes, the __buildDTrace__ task will run the Makefile and that will, in turn, generate the library in <build>/i386 folder.